extensions/copy-gcp-icons/package.json

```
@@ -0,0 +1,58 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "copy-gcp-icons",
+  "title": "Copy GCP Icons",
+  "description": "Copy Product Icons from Google Cloud Platform to Clipboard.",
+  "icon": "icon.png",
+  "author": "pomdtr",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "copy-gcp-icon",
+      "title": "Copy GCP Icon",
```

```suggestion
      "title": "Copy Icon",
```

GCP is redundant since it's already added in the subtitle